tests/test_pytest_cpp.py

```diff
@@ -628,6 +628,10 @@ def test_catch2_failure(exes):
         )
         assert len(failures) == 2
 
+        # test exceptions
+        failures, _ = facade.run_test(exes.get(f"catch2_error{suffix}"), "Error")
+        assert len(failures) == 1
```

Comment:
Should we test that the failure contains the `a runtime error` message?

Reply:
done!